### DISCUSSION TITLE
Default view

### DISCUSSION THREAD

**Discussion by k-i-r-i-k** [nroduit/Weasis#448] [General]
Hello, my dear friends! How to make a default 1 x2 View and 2 monitors in Weasis on Windows?

**Comment by nroduit** [comment]
Hanging protocols are in the feature request but I cannot tell you when it will be available.

**Reply by JPashazade** [reply]
Can you prioritize adding this feature? We are aware that there are many new features requested by users, and we also know that you are developing this application as a single person. However, it appears that there is a significant demand for this feature (default window layout) as well. Would you take this into consideration?

**Comment by rvantklooster** [comment]
We often use Weasis to show DICOM images + all DICOM headers. For us it would be nice to have the DICOM information view right away + all tags when opening Weasis (saves some clicks). Feel free to contact me if you want to discuss details.

**Reply by macgarnigle** [reply]
In the Weasis commands, you can use dcmview2d:layout with flags to set number of views and layout, but I can't find anything on layout identifiers. I want to do the same and open an image in the first pane and the DICOM information in the second.
